manifest.json

```diff
@@ -1,10 +1,14 @@
 {
+  "manifest_version": 2,
   "name": "Switch to Tab",
   "description": "Use the omnibox to search your tabs!",
-  "version": "3",
+  "version": "1",

```

Comment:
Didn't you mean to make this `4`?

Reply:
I set this to 4 in the following commit. But I am not able to figure out how to include that in the patch. This is surprisingly hard for me… sorry :(

I can send another patch request if you'd like.